config/dashboards/dashboards_config.go

```diff
@@ -10,6 +10,8 @@ import (
 )
 
 const (
+	// Namespace annotation for dashboard templates
+	DashboardTemplateAnnotation string = "dashboards.kiali.io/templates"
```

Comment:
[Standard godoc convention](https://blog.golang.org/godoc) is that the comment should be a complete sentence that `begins with the name of the element it describes.` So this should be something like:

```suggestion
	// DashboardTemplateAnnotation is the key name of the namespace annotation for dashboard templates
	DashboardTemplateAnnotation string = "dashboards.kiali.io/templates"
```